src/app/workflow/workflow.component.ts

```diff
@@ -309,8 +299,24 @@ export class WorkflowComponent extends Entry {
    */
   onSelectedVersionChange(version: WorkflowVersion): void {
     this.selectedVersion = version;
-    const currentWorkflowPath = (this.router.url).split(':')[0];
-    this.location.go(currentWorkflowPath + ':' + this.selectedVersion.name);
+    this.updateUrl();
     this.setupFireCloudUrl(this.workflow);
   }
+
+  updateUrl(): void {
```

Comment:
This seems to be a copy paste of https://github.com/dockstore/dockstore-ui2/pull/243/files/bb29fdeb03a62f4500741f45dcf4ac1cb79c5d78#diff-c6c9a9d6217960bf663c6d305d6f7ca0R289
Might be able to avoid by parameterizing my-tools/my-workflows and workflows/containers

At minimum need a link between the two to ensure that one is not updated withoout considering the other

Reply:
Sorry @agduncan94  some odd glitch with killed this comment in my first review

Reply:
I considered that parameterization but wasn't sure if it was overkill. I'll make that change though.